packages/application-extension/schema/commands.json

```diff
@@ -39,13 +39,193 @@
     },
     {
       "command": "application:toggle-right-area",
-      "keys": [""],
+      "keys": ["Accel Shift B"],
```

Comment:
This shortcut is already used to open the 'running kernels' sidepanel:
https://github.com/jupyterlab/jupyterlab/blob/897418f0f25ffbbee3aa06a62770a53246823f85/packages/running-extension/schema/plugin.json#L56-L58